exercises/practice/pov/pov_test.go

```diff
@@ -1,260 +1,156 @@
 package pov
 
 import (
-	"reflect"
 	"sort"
 	"testing"
 )
 
-type arc struct{ fr, to string }
+// auxiliary functions
```

Comment:
I think this comment is not needed because its clear from the file structure and what is exported and not exported, that these are the helpers. 

Besides that, I would recommend to put the helpers either below the place where they are used or below all of the `Test` functions in case they are used in multiple ones. This top down sorting of functions is more consistent with the code we have in the other exercises I think.

Reply:
I have put all the helpers below all of the `Test` functions - I think that may help students who are looking at this file.